includes/class-amp-travel-taxonomies.php

```diff
@@ -0,0 +1,190 @@
+<?php
+/**
+ * Class for Travel taxonomies.
+ *
+ * @package WPAMPTheme
+ */
+
+/**
+ * Class Travel_Taxonomies
+ *
+ * @package WPAMPTheme
+ */
+class AMP_Travel_Taxonomies {
+
+	/**
+	 * AMP_Travel_Taxonomies constructor.
+	 */
+	public function __construct() {
+		add_action( 'init', array( $this, 'register_taxonomies' ) );
+		add_action( 'init', array( $this, 'register_activity_meta' ) );
+		add_action( 'activity_add_form_fields', array( $this, 'add_activity_meta_fields' ) );
+		add_action( 'activity_edit_form_fields', array( $this, 'edit_activity_meta_fields' ) );
+		add_action( 'edit_activity', array( $this, 'save_activity_svg' ) );
+		add_action( 'create_activity', array( $this, 'save_activity_svg' ) );
+	}
+
+	/**
+	 * Register Activity meta.
+	 */
+	public function register_activity_meta() {
+		$args = array(
+			'sanitize_callback' => array( $this, 'sanitize_activity_svg' ),
+			'type'              => 'string',
+			'single'            => true,
+			'show_in_rest'      => true,
+		);
+		register_meta( 'term', 'amp_travel_activity_svg', $args );
+	}
+
+	/**
+	 * Add metafield to Add Activity form page.
+	 */
+	public function add_activity_meta_fields() {
+		?>
+		<div class='form-field form-required term-svg-wrap'>
+			<label for='travel-activity-svg'>SVG</label>
+			<?php wp_nonce_field( basename( __FILE__ ), 'travel_activity_svg_nonce' ); ?>
+			<textarea aria-required='true' name='travel-activity-svg' id='travel-activity-svg'></textarea>
+			<p class="description"><?php esc_attr_e( 'This is for the background icon of the activity term.', 'travel' ); ?></p>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Sanitize the SVG field.
+	 *
+	 * @param string $text Text.
+	 * @return string
+	 */
+	public function sanitize_activity_svg( $text ) {
+		$allowed_html = array(
+			'path' => array(
+				'fill' => true,
+				'd'    => true,
+			),
+			'svg'  => array(
+				'class'   => true,
+				'viewbox' => true,
+			),
+		);
+
+		// Replace ' with " since ' might cause issues with sanitizing.
+		$text = trim( str_replace( "'", '"', $text ) );
+		return wp_kses( $text, $allowed_html );
+	}
+
+	/**
+	 * Add SVG field to Activity term edit view.
+	 *
+	 * @param WP_Term $term Term object.
+	 */
+	public function edit_activity_meta_fields( $term ) {
+		$value = get_term_meta( $term->term_id, 'amp_travel_activity_svg', true );
+		if ( empty( $value ) ) {
+			$value = '';
+		}
+		?>
+		<tr class="form-field term-svg-wrap">
+			<th scope="row"><label for="travel-activity-svg"><?php esc_attr_e( 'SVG', 'travel' ); ?></label></th>
+			<td>
+				<?php wp_nonce_field( basename( __FILE__ ), 'travel_activity_svg_nonce' ); ?>
+				<textarea aria-required="true" name="travel-activity-svg" id="travel-activity-svg">
+				<?php
+				// @codingStandardsIgnoreLine
+				echo $this->sanitize_activity_svg( $value );
+				?>
+			</textarea>
+				<p class="description"><?php esc_attr_e( 'This is for the background icon of the activity term.', 'travel' ); ?></p>
+			</td>
```

Comment:
Could you please change this from `esc_attr_e()` to `esc_html_e()`

Reply:
Thanks, this [looks good](https://github.com/xwp/travel/pull/30/files/62760e5ad56f2ad229afc0811d2e6405ba0a4387..551cfbebeaae56614941f758e128a5f4d88ac549#diff-fc2b02976fb8801660a2625f5a06358dR49).